core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4853,7 +4853,7 @@ public String toString() {
       new Builder(Name.USER_STREAMING_DATA_READ_TIMEOUT)
           .setAlias("alluxio.user.network.data.timeout.ms", Name.USER_NETWORK_DATA_TIMEOUT,
               Name.USER_STREAMING_DATA_TIMEOUT)
-          .setDefaultValue("1h")
+          .setDefaultValue("1m")
```

Comment:
can we be a little more conservative? I would propose 5m? users have very different ufs set ups.